source/api_documentation/transactions_api/_intro.rst

```diff
@@ -20,10 +20,10 @@ The API takes the following optional query parameters:
     - Description
```

Comment:
This one file is referenced in https://github.com/Invoca/developer-docs/blob/e65e767b6a9f47c6ccac19f2a530f57b0142a2d9/source/api_documentation/transactions_api/network_user.rst?plain=1#L30, https://github.com/Invoca/developer-docs/blob/e65e767b6a9f47c6ccac19f2a530f57b0142a2d9/source/api_documentation/transactions_api/affiliate_user.rst?plain=1#L30 and https://github.com/Invoca/developer-docs/blob/e65e767b6a9f47c6ccac19f2a530f57b0142a2d9/source/api_documentation/transactions_api/advertiser_user.rst?plain=1#L30 so it should display for all those pages.